### PR TITLE
add more salt to the game

### PR DESCRIPTION
**Pull request by terranaut1**
## About The Pull Request

Adds salt to the map beyond the few salt-shakers publically available. 

## Why It's Good For The Game

Useful if chemistry blew up, is extremely incompetent and territorial or the traitor CMO killed everyone and is now spreading some AIDS you literally can't cure because salt is rarer than gold. Use pure or turn into saline-glucose solution. Also makes Charcoal ghetto-craftable allowing to deal with Toxins damage after you breathe in a healthy dose of Plasma.
tl;dr more options for ghettochem

## Changelog
:cl: Terranaut

add: Added 3u of Sodium Chloride to the Cup Ramen purchasable from vending machines

/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
Don't chips and such already have salt in them?

**Comment by terranaut1** [comment]
> Don't chips and such already have salt in them?

I was under the impression that the raisins do but grinding them with water one time didn't result in saline-glucose. Checking the code now reveals they do and it should. I'm probably just an idiot but I'm committed to this now.

**Comment by Teepeepee** [comment]
it makes sense anyways, irl cup noodles have a shitton of salt in them

**Comment by Angustmeta** [comment]
Also beef jerky has 2u.

**Comment by improvedname** [comment]
Your saying the vendor with 5 salt shakers of 20 units salt wasnt enough for ghetto crafting, geez how much poison do you drink?

**Review by SpaceManiac — APPROVED** [review]
